tools/antithesis-go-instrumentor/cmd/cmd_files.go

```diff
@@ -145,9 +151,28 @@ func (cfx *CommandFiles) WrapUp() {
 	}
 
 	notifierModule := common.FullNotifierName(cfx.filesHash)
-
-	common.AddDependencies(cfx.inputDirectory, cfx.customerDirectory, cfx.instrumentorVersion, notifierModule)
-	cfx.logWriter.Printf("Antithesis dependencies added to %s/go.mod", cfx.customerDirectory)
+	notifierRelPath := ".."
+	common.AddDependencies(cfx.inputDirectory, cfx.customerDirectory, cfx.instrumentorVersion, notifierModule, notifierRelPath)
+
+	someOffset := ""
+	pathSep := string(os.PathSeparator)
+
+	for modFolder, used := range cfx.dependentModules {
+		if used {
+			relFolders := []string{notifierRelPath}
+			someOffset = common.PathFromBaseDirectory(cfx.inputDirectory, modFolder)
+			if someOffset != "" {
+				num_parents := len(strings.Split(someOffset, pathSep))
```

Comment:
should probably use https://pkg.go.dev/path#Split instead of strings.  Just incase there is an escaped path seperator in a file/directory name.

Reply:
We need to know the number of directory levels to navigate to from modFolder, back to the antithesis module that will be required. 

Example:
`../../../../../notifier` evaluated from `/work/customer/aa/bb/cc/dd` should resolve to `/work/notifier`

In this case, we need to know how many relative levels to 'go up', from the `dd` directory, to refer to the notifier module.  

By using strings.Split (respecting the platform specific PathSeparator) we can readily get the number of levels needed.

Using `path.Split()` or `filepath.Split()` only parses the basename and the directory.  We would have to repeatedly parse the directory to see how many levels there are.  The hope is that  filepath.Split() will properly handle filenames that contain PathSeparator characters.  Is it worth it?

Reply:
```go
	f, _ := filepath.Rel("/test/t/x/y", "/fd")
	fmt.Println(f)
```

Reply:
Done: "tastes great, less filling"

```
someOffset = common.PathFromBaseDirectory(cfx.inputDirectory, modFolder)
if someOffset != "" {
	destModuleFolder := filepath.Join(cfx.customerDirectory, someOffset)
	os.MkdirAll(destModuleFolder, 0777)

	basePath := filepath.Join(cfx.customerDirectory, someOffset)
	targPath := cfx.notifierDirectory
	if altDestModuleFolder, erx := filepath.Rel(basePath, targPath); erx == nil {
		common.AddDependencies(modFolder, destModuleFolder, cfx.instrumentorVersion, notifierModule, altDestModuleFolder)
	}
}

```